Sources/ArcGISToolkit/Components/Augmented Reality/Utilities/WorldTrackingSceneView.swift

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#if !os(visionOS)
```

Comment:
Could `WorldTrackingSceneView` be marked as unavailable on visionOS instead?

Reply:
Same response as [here](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/875/files#r1768811353).